packages/runtime/src/helpers/files.ts

```diff
@@ -1,8 +1,9 @@
 import { cpus } from 'os'
 
+import type { Blobs } from '@netlify/blobs/dist/src/main'
```

Comment:
Why can't this be imported from the package root?

Reply:
Addressed in https://github.com/netlify/next-runtime/pull/2287/commits/239e0443833e1fd4ba9a46d69ebae9c045099ba1